app/controllers/form_controller.rb

```diff
@@ -13,7 +13,11 @@ def submit_form
       mandatory_questions_with_no_response = mandatory_questions_with_no_response(responses_for_page)
 
       if mandatory_questions_with_no_response.empty? && @log.update(responses_for_page.merge(updated_by: current_user))
-        flash[:notice] = "You have successfully updated #{@page.questions.map(&:check_answer_label).reject { |label| label.to_s.empty? }.first&.downcase}" if previous_interruption_screen_page_id.present?
+        if previous_interruption_screen_page_id.present?
+          updated_question = @page.questions.reject { |question| question.check_answer_label.blank? }.first
```

Comment:
are there any soft validation questions that don't have question numbers? Think this'll still be fine in that case but maybe worth regression testing the copy looks right if that's relevant

Reply:
I think if there are no numbers, it would just have the check answers label